interface/forms/newGroupEncounter/common.php

```diff
@@ -279,7 +279,7 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
  if ($sensitivities && count($sensitivities)) {
   usort($sensitivities, "sensitivity_compare");
 ?>
-     <td class='bold' nowrap><?php echo xlt('Sensitivity'); ?>:</td>
+     <td class='bold' nowrap><?php echo xlt('Sensitivity:'); ?>:</td>
```

Comment:
We don't include semicolons in the translation (note there is a colon already there outside the translation) since this causes many constant duplicates. Let me know if this is an issue for RTL, though.